mkt/settings.py

```diff
@@ -1001,8 +1001,13 @@ def JINJA_CONFIG():
 READ_ONLY = False
 
 REDIS_BACKENDS = {
```

Comment:
Does the dict-like syntax works now with 1.6 ? If it does, let's convert everything now and then we don't have to have the single string parser ?